full-service/src/db/txo.rs

```diff
@@ -462,10 +470,14 @@ impl TxoModel for Txo {
 
     fn update_as_received(
         &self,
-        received_account_id_hex: &str,
-        received_subaddress_index: Option<u64>,
+        subaddress_index: Option<u64>,
         received_key_image: Option<KeyImage>,
         block_index: u64,
+        account_id_hex: &str,
+        amount: Amount,
+        target_key: &Vec<u8>,
+        public_key: &Vec<u8>,
+        e_fog_hint: &Vec<u8>,
```

Comment:
Perhaps we leave these not updated for now until we understand better why they are passing tests without being protobuf encoded.